src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -48,7 +48,7 @@ public class TenantServiceImpl implements TenantService {
   private static final String TENANTS_IDS_NOT_MATCHED_ERROR_MSG = "Request body tenantId and path param tenantId should be identical";
   private static final String TENANT_HAS_ACTIVE_USER_ASSOCIATIONS_ERROR_MSG = "Cannot delete tenant with ID {tenantId} because it has an association with a user. "
       + "Please remove the user association before attempting to delete the tenant.";
-  private static final String DUMMY_USERNAME = "*-dummy_user-*";
+  private static final String DUMMY_USERNAME = "dummy_user";
```

Comment:
it's still better to keep smth that noone will type, dummy_user can be used by devs or qas :)

Reply:
we basically not creating any dummy user here, just record in user_tenant table, also because now we are not storing in central tenant so we will not have any collision for login, so I think we are ok here:)

Reply:
ah right, if we skip central tenant, it's true